styles.css

```diff
@@ -0,0 +1,1007 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@100&family=Roboto&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  width: 1100px;
+  margin-left: 100px;
+  background-color: #e5e5e5;
+}
+
+.grid {
+  display: grid;
+  grid-template-rows: 20% 40% 23% 17%;
+}
+
+.home {
+  color: #fff;
+  width: 1100px;
+}
+
+header {
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  background-color: #3e3c3c;
+  display: flex;
+  justify-content: space-around;
+  height: 50px;
+}
+
+.name {
+  color: #fff;
+  margin-top: 15px;
+  margin-bottom: 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  font-weight: 900;
+  height: 40px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.navigator {
+  width: 200px;
+  height: 40px;
+  margin-top: 10px;
+  display: inline-flex;
+  align-items: center;
+}
+
+.portfolio {
+  color: #fff;
+  height: 20px;
+  width: 63px;
+  left: 12px;
```

Comment:
Hi, Henry. Thank you for your suggestions.  But I can't follow some of them because of project requirements. Let's say here I am actually following Figma's conditions using the Inter font. Please note that in the mobile form, Roboto is required, but in the desktop version, it is Inter.

Reply:
Thank you, @Gegardus, for your question. You can get the inter font in google font.
Check it out here => [Inter font](https://fonts.google.com/specimen/Inter)

Reply:
Thank you, Henry, for your hint, but I already implemented Inter font. You can check it in my CSS file at the very first line. 
In my comment, I just mean, that despite your kindly suggestion, I have implemented the right font here.

Reply:
Oh! Good job👍 just make sure that the "portfolio", "About", and "Contact" texts maintain the same font. 
We are together in this💪

Reply:
Done.